scopes/dependencies/dependency-resolver/dependency-linker.ts

```diff
@@ -492,55 +480,93 @@ export class DependencyLinker {
     }
   }
 
-  private linkHarmony(dirMap: ComponentMap<string>, rootDir: string): LinkDetail | undefined {
-    const name = 'harmony';
-    const packageName = '@teambit/harmony';
+  private removeSymlinkTarget(targetPath: string, hasLocalInstallation = false) {
+    // TODO: change to fs.lstatSync(dest, {throwIfNoEntry: false});
+    // TODO: this requires to upgrade node to v15.3.0 to have the throwIfNoEntry property (maybe upgrade fs-extra will work as well)
```

Comment:
maybe we can check if it exists and use it, otherwise use the old code ? this way we don't force people to use a recent version of node

Reply:
seems like the code doesn't force users to use the new node, only suggest to change it in the future. it makes sense to avoid if/else node version and use the method that works for both